tendrl/common/manager/manager.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         self,
         name,
         integration_id,
+        node_id,
```

Comment:
So the Base manager class is common and node_agent, gluster_integration and ceph_integration managers extend from the same. Also the rpc job processor is common code.

The parameter node_ids is not a mandatory one. If its passed, current node_id is checked against the same. We dont have module specific rpc job processors anymore after refactoring, so this is required and I dont see an issue. Even in case of gluster and ceph integration, if required jobs can be targeted to specific set of nodes (like to mons in case of ceph cluster).